cmake/zlib.cmake

```diff
@@ -37,6 +37,8 @@ MACRO (MYSQL_CHECK_ZLIB_WITH_COMPRESS)
   IF(WITH_ZLIB STREQUAL "bundled")
     MYSQL_USE_BUNDLED_ZLIB()
   ELSE()
+    FIND_PACKAGE(PkgConfig QUIET)
+    PKG_GET_VARIABLE(ZLIB_ROOT zlib prefix)
     INCLUDE(FindZLIB)
     IF(ZLIB_FOUND)
      INCLUDE(CheckFunctionExists)
```

Comment:
Is  `SET(CMAKE_REQUIRED_LIBRARIES ${ZLIB_LIBRARIES})`  required below?

Reply:
Yes - thanks again for pointing this out